picocom.c

```diff
@@ -1813,6 +1821,25 @@ parse_args(int argc, char *argv[])
                 r = -1;
             }
             break;
+	case 'R':
+	    {
```

Comment:
Instead of 'rate', I'd suggest '-T' or '--tx-delay' (in us) as mentioned in issue #96. One advantage is that the numbers to be used are smaller and. thus, less to type for the user. Compate '-T1' to '-R1000000'. Easy to confuse the numbers of zeroes there. And also...